contrib/drivers/pgsql/pgsql_convert.go

```diff
@@ -128,7 +128,7 @@ func (d *Driver) ConvertValueForLocal(ctx context.Context, fieldType string, fie
 		if err := result.Scan(fieldValue); err != nil {
 			return nil, err
 		}
-		return result, nil
+		return []string(result), nil
```

Comment:
The `pq.StringArray` is type of `[]string`, the `[]string(result)` is `[][]string`, you sure?

Reply:
Sure, []string(result) is only converting pq.ArrayString to the base type []string, not [][]string. As a result, when calling gconv.Strings, it directly matches the case []string branch instead of relying on originValueAndKind := reflection.OriginValueAndKind(any).

Reply:
OK, I see.